pkg/flag/db_flags.go

```diff
@@ -50,17 +50,17 @@ var (
 		ConfigName: "db.no-progress",
 		Usage:      "suppress progress bar",
 	}
-	DBRepositoryFlag = Flag[string]{
+	DBRepositoryFlag = Flag[[]string]{
 		Name:       "db-repository",
 		ConfigName: "db.repository",
-		Default:    db.DefaultRepository,
-		Usage:      "OCI repository to retrieve trivy-db from",
+		Default:    []string{db.DefaultGHCRRepository, db.DefaultECRRepository},
```

Comment:
We're unsure how the free tier in ECR Public works, so we should probably avoid adding ECR for now. Instead, we'll document how to use ECR Public in another PR.